apps/atomy-q/API/app/Jobs/ProcessQuoteSubmissionJob.php

```
@@ -47,11 +47,17 @@ public function handle(QuoteIngestionOrchestrator $orchestrator): void
     public function failed(\Throwable $exception): void
     {
         $submission = QuoteSubmission::find($this->quoteSubmissionId);
-        
+
+        Log::error('ProcessQuoteSubmissionJob exhausted retries', [
+            'quote_submission_id' => $this->quoteSubmissionId,
+            'error_class' => $exception::class,
+            'error_message' => $exception->getMessage(),
+        ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'failed'. (undefined)

(StaticAccess)